### PR TITLE
ENH Update reference to supported modules data

### PR DESCRIPTION
**Pull request by GuySartorelli**
Needs https://github.com/silverstripe/supported-modules/pull/29

Also update CI to use gha-ci which is required both for using a correct PHP version and making sure patches get auto-tagged

## Issues
- https://github.com/silverstripe/.github/issues/237

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.1",
+        "php": "^8.1",
```
Comment (ChloeSartorelli) [inline composer.json:15]: supported-modules needs 8.1

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -4,30 +4,8 @@ on:
   push:
   pull_request:
   workflow_dispatch:
-  # Every Tuesday at 2:00pm UTC
-  schedule:
```
Comment (emteknetnz) [inline .github/workflows/ci.yml:8]: We should keep the schedule? If an auto-tag fails on merge (happens very occasionally) the cron will pick it up

**Review by emteknetnz — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (ChloeSartorelli) [reply .github/workflows/ci.yml:8]: Standardiser will add dispatch-ci to this module when it's next run. This is the way we do this for all repos that use standard CI.

**Review by emteknetnz — APPROVED** [review]

**Comment by emteknetnz** [comment]
Have released [1.2.0](https://github.com/silverstripe/silverstripe-tx-translator/releases/tag/1.2.0)

**Pull request merged**